tests/test_acl.py

```diff
@@ -70,22 +96,70 @@ def test_AclTableCreation(self, dvs):
                     assert True
                 else:
                     assert False
-    
+
         assert set(member_groups) == set(acl_table_groups)
-    
-        # check port binding 
+
+    def verify_acl_port_binding(self, dvs, adb, expt, bind_ports):
+        atbl = swsscommon.Table(adb, "ASIC_STATE:SAI_OBJECT_TYPE_ACL_TABLE_GROUP")
+        acl_table_groups = atbl.getKeys()
+        assert len(acl_table_groups) == expt
+
         atbl = swsscommon.Table(adb, "ASIC_STATE:SAI_OBJECT_TYPE_PORT")
-    
         port_groups = []
         for p in [dvs.asicdb.portnamemap[portname] for portname in bind_ports]:
             (status, fvs) = atbl.get(p)
             for fv in fvs:
                 if fv[0] == "SAI_PORT_ATTR_INGRESS_ACL":
                     assert fv[1] in acl_table_groups
                     port_groups.append(fv[1])
-    
+
+        assert len(port_groups) == expt
         assert set(port_groups) == set(acl_table_groups)
-    
+
+    def verify_acl_lag_binding(self, adb, expt):
```

Comment:
>expt [](start = 42, length = 4)

change to bind_ports as well, do not need expt here.

Reply:
for lag case it different with port case, bind_ports len may not equal to the binding lag number if  some LAG not configured yet, so I think we still need a expt? and also, I didn't find out how a lag port name 
 was mapped to a  SAI_OBJECT_TYPE_LAG_MEMBER in ASIC DB,  unless you create one LAG and then know that the only SAI_OBJECT_TYPE_LAG_MEMBER record in  ASIC DB is yours?